composer.json

```diff
@@ -10,20 +36,20 @@
   "require": {
     "dompdf/dompdf" : "0.7.*",
     "electrolinux/phpquery": "^0.9.6",
-    "symfony/config": "~2.5.0",
-    "symfony/dependency-injection": "~2.5.0",
-    "symfony/event-dispatcher": "~2.5.0",
-    "symfony/filesystem": "~2.5.0",
-    "symfony/process": "~2.5.0",
-    "psr/log": "1.0.0",
-    "symfony/finder": "~2.5.0",
+    "symfony/config": "^2.5.0",
```

Comment:
Well `~2.5` would allow 2.8 because there isn't the last `.0` on there. We probably actually should do `~2.5 || ~3.0` to be compatible with Drupal 8.4.x but I haven't tested with 8.4.x yet